.github/workflows/cd-deploy-www-staging.yml

```diff
@@ -3,10 +3,19 @@ on:
   push:
     branches:
       - master
+    paths:
+      - 'sites/website/**'
```

Comment:
Is it possible to also detect when any markdown file or api-extractor.json file changes? Any changes in these technically also would effect the content on the documentation site?

Reply:
Updated for www to include any markdown changes using `**.md` as referenced from https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#patterns-to-match-file-paths cheat sheet.